cluster/cluster.go

```diff
@@ -348,7 +357,9 @@ func (c *Cluster) baseCreateStackInput() *cloudformation.CreateStackInput {
   ]
 }
 `),
+		}
 	}
+	return creq
```

Comment:
Almost nit but something like:

```go
var stackPolicyBody string
if c.Experimental.ExternalEtcd.Enabled {
  // We don't manage an etcd cluster in this stack(therefore `ExternalEtcd`), hence no need to lock
  stackPolicyBody = "{}"
} else {
  // Internally managed etcd cluster isn't updatable for now.
  // See https://github.com/coreos/coreos-kubernetes/pull/544#issuecomment-247483015
  stackPolicyBody = `{
    "Statement" : [
      {
        "Effect" : "Deny",
       "Action" : "Update:*",
       "Principal" : "*",
       "Resource" : "LogicalResourceId/InstanceEtcd*"
     },
     {
        "Effect" : "Allow",
        "Principal" : "*",
        "Action" : "Update:*",
        "Resource" : "*"
      }
    ]
  }
`
}

return &cloudformation.CreateStackInput{
    StackName:    aws.String(c.ClusterName),
    OnFailure:    aws.String(cloudformation.OnFailureDoNothing),
    Capabilities: []*string{aws.String(cloudformation.CapabilityCapabilityIam)},
    Tags:         tags,
    StackPolicyBody: aws.String(stackPolicyBody),
}
```

would be better because then we can be explicit about the only difference is `StackPolicyBody`?